pre-steps/3_add_items.ts

```diff
@@ -53,3 +53,5 @@ async function addItems() {
     `     https://explorer.solana.com/tx/${response.signature}?cluster=devnet`
```

Comment:
If other developers want to mint an NFT collection on Solana **mainnet**, all the _Solana Explorer_ links are going to break, because they're currently set to `cluster=devnet`. Let's state clearly in the docs that this codebase is designed for _devnet_ mints only for the moment. :wink: